packages/core/content-type-builder/admin/src/pages/App/index.tsx

```diff
@@ -32,20 +31,20 @@ const App = () => {
     }
   }, []);
 
-  // FIXME Error here
   return (
     <Page.Protect permissions={PERMISSIONS.main}>
       <Page.Title>{title}</Page.Title>
       <AutoReloadOverlayBlockerProvider>
         <FormModalNavigationProvider>
-          {/* eslint-disable-next-line @typescript-eslint/ban-ts-comment */}
-          {/* @ts-ignore */}
           <DataManagerProvider>
             <Layouts.Root sideNav={<ContentTypeBuilderNav />}>
               <Suspense fallback={<Page.Loading />}>
                 <Routes>
                   <Route path="content-types/:uid" element={<ListView />} />
-                  <Route path={`component-categories/:categoryUid/*`} element={<RecursivePath />} />
+                  <Route
+                    path={`component-categories/:categoryUid/:componentUid`}
+                    element={<ListView />}
+                  />
```

Comment:
This is neat because fewer splat routes means the react router v7 migration [will be easier](https://reactrouter.com/upgrading/v6#v7_relativesplatpath)